src/DivaView.js

```diff
@@ -38,16 +38,18 @@ class DivaView {
   /**
    * Called when the visible page changes in the diva.js viewer.
    * @param {number | number[]} pageIndexes - The zero-index or -indexes of the page(s) visible.
+   * @param {boolean} delay - whether to delay the loading of the page. defaults to true
```

Comment:
Since this is optional with a default value, the correct way to represent this with jsdoc would be `@param {boolean} [delay=true]`.